src/ui/view-dispatch.ts

```diff
@@ -0,0 +1,36 @@
+import { html, customElement, css } from 'lit-element'
+import { Connected } from './connected'
+import { sharedStyles } from './shared-styles'
+import { dispatch } from '../state'
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'view-dispatch': ViewDispatchElement
+  }
+}
+
+@customElement('view-dispatch')
+export class ViewDispatchElement extends Connected {
+
+
+  dispatchExample(_e: Event) {
+    dispatch.mymodel.takeValue(25);
+  }
+  render() {
+    return html`<button @click=${this.dispatchExample}>Click me to ask back end to square 25</button>
+     then click <a href="/subscribe">here</a> to view result;
+    
+    `
+  }
```

Comment:
It would be more useful to have something a little 'richer', such as an input to enter a value and the button then provides the call to square it.